.travis.yml

```diff
@@ -1,14 +1,34 @@
 language: python
 python:
+  - "3.4"
+  - "3.3"
   - "2.7"
   - "2.6"
 env:
   - DJANGO=1.3
   - DJANGO=1.4
+  - DJANGO=1.5
+  - DJANGO=1.6
+  - DJANGO=1.7
+  - DJANGO=1.8
+matrix:
+  exclude:
+    - python: "2.6"
+      env: DJANGO=1.7
+    - python: "2.6"
+      env: DJANGO=1.8
+    - python: "3.4"
+      env: DJANGO=1.4
+    - python: "3.3"
+      env: DJANGO=1.4
+    - python: "3.4"
+      env: DJANGO=1.3
+    - python: "3.3"
+      env: DJANGO=1.3
 install:
   - pip install -q Django==$DJANGO --use-mirrors
   - pip install pep8 --use-mirrors
-  - pip install https://github.com/dcramer/pyflakes/tarball/master
+  - pip install pyflakes
```

Comment:
Why aren't we using `--use-mirrors` here too, now? This can go on the same line as pep8 though, anyway..

We could use flake8 too.